diff --git a/driver/src/common/xaie_helper.c b/driver/src/common/xaie_helper.c
index 071d4e3b..3161c02e 100644
--- a/driver/src/common/xaie_helper.c
+++ b/driver/src/common/xaie_helper.c
@@ -2858,7 +2858,7 @@ AieRC XAie_Txn_MergeSync(XAie_DevInst *DevInst, u8 num_tokens, u8 num_cols)
 				}
 		}
 
-		u32* tctDataBuff = (u32 *)calloc(sizeof(tct_op_t), 1);
+		u32* tctDataBuff = (u32 *)calloc(1, sizeof(tct_op_t));
 		tctDataBuff[0] = ((0x000000FF & num_tokens) || (0x0000FF00 & (num_cols << 8)));
 		TxnInst->CmdBuf[TxnInst->NumCmds].Opcode = XAIE_IO_CUSTOM_OP_MERGE_SYNC;
 		TxnInst->CmdBuf[TxnInst->NumCmds].Size = (u32)sizeof(tct_op_t);
@@ -2914,7 +2914,7 @@ AieRC XAie_Txn_DdrAddressPatch(XAie_DevInst *DevInst, u64 regaddr, u64 argidx,
 				}
 		}
 
-		patch_op_t* patchDataBuff = (patch_op_t *)calloc(sizeof(patch_op_t), 1);
+		patch_op_t* patchDataBuff = (patch_op_t *)calloc(1, sizeof(patch_op_t));
 		patchDataBuff->regaddr = regaddr;
 		patchDataBuff->argidx = argidx;
 		patchDataBuff->argplus = argplus;
